java/client/src/main/java/glide/api/commands/GeospatialIndicesBaseCommands.java

```diff
@@ -78,4 +80,58 @@ CompletableFuture<Long> geoadd(
      * }</pre>
      */
     CompletableFuture<Double[][]> geopos(String key, String[] members);
+
+    /**
+     * Returns the distance between <code>member1</code> and <code>member2</code> in the geospatial
+     * index stored at <code>key</code>.
+     *
+     * @see <a href="https://valkey.io/commands/geodist">valkey.io</a> for more details.
+     * @param key The key of the sorted set.
+     * @param member1: The name of the first member
+     * @param member2: The name of the second member
+     * @param geoUnit: The unit of distance measurement. See {@link GeoUnit}. If not specified, the
```

Comment:
Skip `see` here - it will be still clickable. Also, it couldn't be `not specified`.
But you can move default section to the second function signature.